test/iterators/bfs.jl

```diff
@@ -0,0 +1,44 @@
+@testset "BFSIterator" begin
```

Comment:
It would be good to test edge cases: empty graph, graph with 2 connected components

Reply:
same